source/backend/api/Areas/Projects/Controllers/ProjectController.cs

```diff
@@ -107,6 +113,29 @@ public async Task<IActionResult> AddProject(ProjectModel projectModel)
             return Ok(newProject);
         }
 
+        /// <summary>
+        /// Updates the project.
+        /// </summary>
+        /// <returns></returns>
+        [HttpPut("{id:long}")]
+        [HasPermission(Permissions.ProjectEdit)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(ProjectModel), 200)]
+        [SwaggerOperation(Tags = new[] { "project" })]
+        public IActionResult UpdateProject([FromRoute]long id, [FromBody] ProjectModel model)
+        {
+            _logger.LogInformation(
```

Comment:
Would be good to come up with some logging standards, as everything is ad-hoc at the moment.

Reply:
This log statement is based off that research spike I did a while back on structured logging best practices. Where we are lacking is in following that for evert controller/method. I also wish there were a way to automate/eliminate this boilerplate

Reply:
removed for now.